crates/ethrpc/src/lib.rs

```diff
@@ -46,13 +46,21 @@ module! {
         pub struct GetBlockByNumber as "eth_getBlockByNumber"
             (BlockSpec, Hydrated) => Option<Block>;
 
-        /// Returns a collection of all logs matching the given filter.
-        pub struct GetLogs as "eth_getLogs"
-            (LogFilter,) => Vec<Log>;
+        /// Returns the number of transactions in a block from a block matching the given block hash.
+        pub struct GetBlockTransactionCountByHash as "eth_getBlockTransactionCountByHash"
+            (Digest,) => Option<u64>;
+
+        /// Returns the number of transactions in a block matching the given block number.
+        pub struct GetBlockTransactionCountByNumber as "eth_getBlockTransactionCountByNumber"
+            (BlockId,) => Option<u64>;
```

Comment:
This methods supports block number or block tag, but not block hash as input. Not sure how we should exclude that from the enum... 

Also wondering if we want to return U256 here instead of u64?

Reply:
This cannot be a `u64` as it will try to deserialise as a number and not a hex-encoded string as specified by the spec.